lua/entities/gmod_wire_expression2/core/utf8.lua

```diff
@@ -0,0 +1,47 @@
+local unpack = unpack
+local isnumber = isnumber
+local math_floor = math.floor
+local utf8 = utf8
+local utf8_char = utf8.char
+local utf8_codepoint = utf8.codepoint
+local utf8_len = utf8.len
+
+__e2setcost(1)
+
+--- Returns the UTF-8 string from the given Unicode code-points.
+e2function string utf8char(...)
+	local args = { ... }
+	local count = #args
+	if count == 0 then return "" end
+	local codepoints = {}
+	for i = 1, count do -- TODO: Limit the count to "max" amount. How about 512?
```

Comment:
Remove this TODO. Limiting is not a good idea.

Reply:
Are you aware of what's happening there tho?

Reply:
Yea, it's not that bad.